snowflake/pkg/utils/utils.go

```diff
@@ -0,0 +1,140 @@
+// Copyright 2022 Antrea Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
```

Comment:
nit: Quan pointed out in the past that `utils` was too generic for a package name. So I wonder if we could move these functions to `pkg/utils/file/file.go` (package can then be imported as `fileutils`).

Reply:
Sounds good, changed.